classes/student.rb

```diff
@@ -1,7 +1,9 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(name, age, parent_permission, classroom)
+  attr_accessor :classroom
+
+  def initialize(name, age, parent_permission, classroom = nil)
```

Comment:
* Kindly add the association `a student belongs to a classroom` by adding `a method that serves as the 'belongs-to' side (a student belongs to a classroom).`  [ Link to the article with examples](
https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Reply:
Implemented